doc/TILE_ON_FRAME_BOUNDARY.md

```diff
@@ -0,0 +1,35 @@
+# Tile on a Frame boundary
+
+## Accessing Tiled-pixels outside of a frame
+
+Tiled input and output planes (type PlaneRegion) are configured in order to
+allow memory access of its pixels that are located outside of frame boundary.
+This configuration is only effective when the tile is adjacent to right or bottom frame boundary and
+a partition block straddles on any frame boundary.
+
+The type names of tiled input and output in the codebase is defined in:
+
+"'TileStateMut' struct"
+"pub struct TileStateMut<'a, T: Pixel> {'
+"  ..."
+"  pub input: &'a Frame<T>,     // the whole frame"
+"  pub rec: TileMut<'a, T>,"
+"  ..."
+"}"
+
+## How bounding box is enforced to a PlaneRegion and now it allows to access outside of frame pixels
+
+A memory access to tiled input and output pixels in a _tile_ 
+is bounded by 'rect' field of each plane (of type 'PlaneRegion') of the 'input_tile' or 'rec' of 'TileStateMut'.
```

Comment:
```suggestion
is bounded by 'rect' field of each plane (of type `PlaneRegion`) of the `input_tile` or `rec` of `TileStateMut`.
```

So it will be better when we render the markdown :)

Reply:
Done. Thanks!